components/customs/multi-select.tsx

```diff
@@ -15,22 +15,23 @@ export type MultiSelectOption = {
 interface MultiSelectProps {
   heading: string;
   options: string[];
+  optionToLabelMap: Record<string, string | undefined>;
   selected: string[];
   handleOnChange: (selected: string[]) => void;
 }
 
-const createMultiSelectOptionList = (options: string[]) => {
+const createMultiSelectOptionList = (options: string[], optionToLabelMap: Record<string, string | undefined>) => {
   return options.map((option: string) => {
     return {
-      label: option,
-      value: option.toLowerCase(),
```

Comment:
Removal of `toLowerCase` was to move it to a place that was more appropriate and made more sense given what this line is actually accomplishing (it's similar to the `trim` so I moved it next to it)